plugins/plugin_test/plugin_test.cpp

```diff
@@ -0,0 +1,80 @@
+/*
+ Copyright 2025 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+#include "plugin_test.h"
+#include <QMenu>
+#include <QMenuBar>
+#include <iostream>
+#include "ui/main_window.h"
+
+PluginTest::PluginTest(QObject* parent) :
+    QObject(parent)
+{
+}
+
+PluginTest::~PluginTest() {}
+
+bool PluginTest::Initialize(MainWindow* main_window)
+{
+    if (!main_window)
+    {
+        return false;
+    }
+
+    QMenu*          help_menu = nullptr;
+    QList<QAction*> menu_bar_actions = main_window->menuBar()->actions();
+    for (QAction* action : menu_bar_actions)
+    {
+        if (action->menu())
+        {
+            if (action->text().trimmed().replace("&", "") == "Help")
+            {
+                help_menu = action->menu();
+                break;
+            }
+        }
+    }
+
+    if (help_menu)
```

Comment:
consider an early return to reduce nesting
```suggestion
    if (!help_menu) return false;
```

Reply:
done